test/float8/test_numerics_integration.py

```diff
@@ -144,10 +145,12 @@ def test_encoder_fw_bw(
             cast_config_grad_output=cast_config_grad_output,
         )
 
-        convert_to_float8_training(
-            model_fp8,
-            config=config,
-        )
+        # convert_to_float8_training(
+        #     model_fp8,
+        #     config=config,
+        # )
+        quantize_(model_fp8, float8_for_training(config))
```

Comment:
this was shorter than i thought it would be :D

Reply:
note that setting the right config values may take longer